### PR TITLE
changing gr to release for windows.ini

### LINKED ISSUE #26: Update README with Python requirements
Currently I believe that mozprofile requires Python 2.7 and fails under 3.x.
Need to verify and make sure we clearly outline requirements in the README.md.

### PR DESCRIPTION
**Pull request by rpappalax**
I noticed we have a few more windows tweaks so, starting a new PR.

Fixes #26

### CODE REVIEW

**Comment by cloud-services-qa** [comment]
Nightly installs no problem!  \o/

Beta path needs some correcting on the install??  I noticed it gets installed to C:\\Program Files\Mozilla Firefox, but we never copy it over to FirefoxBeta or something like that, so I think it doesn't know how to launch....


pdehaan@thats-so-lenovo ~/git/ff-tool
$ ff -c beta
mv: target 'Beta' is not a directory

----------------------------------------
LOADING configs\windows.ini
----------------------------------------

Downloading beta to _temp\browsers\FirefoxBeta.exe
[localhost] local: chmod +x _temp\browsers\FirefoxBeta.exe
[localhost] local: _temp\browsers\FirefoxBeta.exe -ms
[localhost] local: mv "C:/"Program Files"/"Mozilla Firefox"" "C:/"Program Files"/"Mozilla Firefox Beta""

Fatal error: local() encountered an error (return code 1) while executing 'mv "C:/"Program Files"/"Mozilla Firefox"" "C:/"Program Files"/"Mozilla Firefox Beta""'

Aborting.
(venv)

**Comment by pdehaan** [comment]
Weird, should be happening in https://github.com/rpappalax/ff-tool/blob/dev/fftool/firefox_install.py#L30
Wonder if it is because of all those quotes

**Comment by cloud-services-qa** [comment]
Dearest @rpappalax,

As we peer deeper into the abyss, things start to get weirder... It appears if I try to manually move the Beta/DevEditoin folder on The Lenovo, I get a "permission denied" error.

You can see the permissions on the files below. Deeper still, we can see that "Beta"+"Developer Edition" have different permissions/owner versus Nightly. This makes we wonder if we installed Nightly at all, or if it was an existing install.

This is where Windows really isn't that spectacular since it forces everything into Program Files so we can't just manage our own separate Firefox versions like we do on OSX/Linux.
I'm starting to think we just scorched earth all the Firefox installs on this Lenovo and try and install evreything using fftool, and scratch our collective heads about what to do about renaming that Firefox Beta folder. It's odd that it appears to be owned by "pdehaan", yet typing `whoami` confirms that we're currently running as that user. Not even sure who I could chown that directory to.

```sh
$ ls -las "/cygdrive/c/Program Files/"
total 149
12 drwxr-x---+ 1 NT SERVICE+TrustedInstaller NT SERVICE+TrustedInstaller   0 Mar 16 20:57 .
 8 d---r-x---+ 1 NT SERVICE+TrustedInstaller NT SERVICE+TrustedInstaller   0 Mar 14 16:06 ..
 4 drwxr-x---+ 1 NT SERVICE+TrustedInstaller NT SERVICE+TrustedInstaller   0 Mar 14 14:40 'Common Files'
 1 -rwxrwx---+ 1 Administrators              SYSTEM                      174 Mar  1 01:01 desktop.ini
 0 drwxr-x---+ 1 Administrators              None                          0 Jul  9  2015 DIFX
 8 drwxrwx---+ 1 Administrators              None                          0 Jul 21  2015 'DisplayLink Core Software'
 8 drwxrwx---+ 1 SYSTEM                      SYSTEM                        0 Jul 16  2015 'Dolby Digital Plus'
12 drwxrwx---+ 1 Administrators              pdehaan                       0 Mar 14 16:45 'Firefox Developer Edition'
 4 drwxrwx---+ 1 SYSTEM                      SYSTEM                        0 Mar 14 14:40 Intel
 0 drwxrwx---+ 1 SYSTEM                      SYSTEM                        0 Mar 14 14:40 'Intel Corporation'
 4 drwxr-x---+ 1 NT SERVICE+TrustedInstaller NT SERVICE+TrustedInstaller   0 Mar  1 03:01 'Internet Explorer'
 4 drwxrwx---+ 1 Administrators              None                          0 Mar 14 14:40 Lenovo
 0 drwxrwx---+ 1 Administrators              None                          0 Jul  9  2015 'Lenovo USB Graphics'
 4 drwxrwx---+ 1 Administrators              None                          0 Jul  9  2015 mcafee
 0 drwxrwx---+ 1 Administrators              None                          0 Jul  9  2015 mcafee.com
16 drwxrwx---+ 1 Administrators              pdehaan                       0 Mar 17 09:47 'Mozilla Firefox'
 0 drwxrwx---+ 1 SYSTEM                      SYSTEM                        0 Mar 14 15:33 MSBuild
32 drwxrwx---+ 1 Administrators              SYSTEM                        0 Mar 17 09:41 Nightly
 4 drwxrwx---+ 1 SYSTEM                      SYSTEM                        0 Feb 15 11:26 nodejs
 0 drwxrwx---+ 1 SYSTEM                      SYSTEM                        0 Jul  9  2015 'Nok Nok Labs'
 0 drwxrwx---+ 1 SYSTEM                      SYSTEM                        0 Mar 14 14:38 Realtek
 0 drwxr-x---+ 1 NT SERVICE+TrustedInstaller NT SERVICE+TrustedInstaller   0 Mar 14 15:33 'Reference Assemblies'
 0 drwxr-x---+ 1 NT SERVICE+TrustedInstaller NT SERVICE+TrustedInstaller   0 Mar  1 01:05 SenseClient
 4 drwxrwx---+ 1 pdehaan                     pdehaan                       0 Oct 13 14:42 Sikuli
 0 drwxrwx---+ 1 SYSTEM                      SYSTEM                        0 Mar 14 14:38 Synaptics
 0 drwxrwx---+ 1 Administrators              SYSTEM                        0 Jul 10  2015 'Uninstall Information'
 8 drwxr-x---+ 1 NT SERVICE+TrustedInstaller NT SERVICE+TrustedInstaller   0 Mar  1 03:01 'Windows Defender'
 4 drwxr-x---+ 1 NT SERVICE+TrustedInstaller NT SERVICE+TrustedInstaller   0 Mar  1 03:06 'Windows Journal'
 4 drwxr-x---+ 1 NT SERVICE+TrustedInstaller NT SERVICE+TrustedInstaller   0 Mar  1 03:01 'Windows Mail'
 4 drwxr-x---+ 1 NT SERVICE+TrustedInstaller NT SERVICE+TrustedInstaller   0 Mar  1 03:01 'Windows Media Player'
 0 drwxr-x---+ 1 NT SERVICE+TrustedInstaller NT SERVICE+TrustedInstaller   0 Mar  1 01:05 'Windows Multimedia Platform'
 0 drwxr-x---+ 1 NT SERVICE+TrustedInstaller NT SERVICE+TrustedInstaller   0 Mar  1 01:05 'Windows NT'
 4 drwxr-x---+ 1 NT SERVICE+TrustedInstaller NT SERVICE+TrustedInstaller   0 Mar  1 03:01 'Windows Photo Viewer'
 0 drwxr-x---+ 1 NT SERVICE+TrustedInstaller NT SERVICE+TrustedInstaller   0 Mar  1 01:05 'Windows Portable Devices'
 0 drwxr-x---+ 1 NT SERVICE+TrustedInstaller NT SERVICE+TrustedInstaller   0 Mar  1 01:05 'Windows Sidebar'
 0 drwxr-x---  1 Unknown+User                Unknown+Group                 0 Mar 16 16:27 WindowsApps
 0 drwxr-x---+ 1 NT SERVICE+TrustedInstaller NT SERVICE+TrustedInstaller   0 Mar  1 01:05 WindowsPowerShell


$ mv "/cygdrive/c/Program Files/Firefox Developer Edition"/ "/cygdrive/c/Program Files/Auroralis"
mv: cannot move '/cygdrive/c/Program Files/Firefox Developer Edition/' to '/cygdrive/c/Program Files/Auroralis': Permission denied


$ mv "c:/Program Files/Firefox Developer Edition"/ "c:/Program Files/Auroralis"
mv: cannot move 'c:/Program Files/Firefox Developer Edition/' to 'c:/Program Files/Auroralis': Permission denied
```

**Comment by pdehaan** [comment]
Just to continue live blogging this for historical purposes...

**TL;DR:** :poop:

I can't figure out why `mv` just fails with little warning, but it seems the solution is to run Cygwin as Administrator (ouch).

And now I've successfully installed Firefox Beta on The Lenovo.
Next failures are:

1. Running `C:/Program Files/Mozilla Firefox Beta/firefox.exe --version` fails via Fabric (apparently). When I run it right in Cygwin directly, it just exits (seemingly with an exit code of `0`, which is cool) but through Fabric it fails. I wrapped the `get_firefox_version()` method in a `try...except` block, which just causes it to silently fail now. Conversely, we could just do an `is_windows()` check and print a prettier error message saying Windows smells. Or maybe we need further research into why it doesn't apparently work on Windows.

2. Now that I have Firefox installed and "working beautifully(tm)", it fails if you try running it a second time, with the error message of:

    > ```
[localhost] local: mv 'C:/Program Files/Mozilla Firefox" "C:/Program Files/Mozilla Firefox Beta"
mv: cannot move 'C:/Program Files/Mozilla Firefox' to 'C:/Program Files/Mozilla Firefox Beta/Mozilla Firefox': Directory not empty.
> ```

    1. First, our pathing seems weird where it's trying to do "Mozilla Firefox Beta/Mozilla Firefox".
    2. Second, what the what??? Even if this did work, we'd still probably have the issue of moving files into an existing directory. Do we need to seriously `rm -rf` the "Mozilla Firefox Beta" folder before trying to install it again?
    3. Potential issue around if Release is already installed and then try and install Beta. In theory, we'd potentially overwrite the existing Firefox "release" channel with beta (since they share target file names) and then we'd rename the folder to "Mozilla Firefox Beta", effectively killing their original Firefox install.

**Comment by cloud-services-qa** [comment]
@pdehaan  - ready for merge?

**Comment by pdehaan** [comment]
r+ :+1:
